tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -259,6 +253,23 @@ def _read_label_file(label_directory: str, segment_name: str, dataset_type: str)
     return label_contents
 
 
+def _read_label_file_10k(label_directory: str, segment_name: str) -> Dict[str, Any]:
+    source_label_contents = []
+    label_filename = os.path.join(
+        label_directory, "pan_seg", "polygons", f"pan_seg_{segment_name}.json"
+    )
+    try:
+        with open(label_filename, "r") as fp:
+            source_label_contents.append(json.load(fp))
+    except FileNotFoundError:
+        error(f"Missing label file pan_seg_{segment_name}.json")
```

Comment:
`error` is used for cli only,
it is ok to let `FileNotFoundError` go, do not catch it